google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/standard/AppEngineStandardDeploymentEditor.java

```diff
@@ -65,13 +65,33 @@ public AppEngineStandardDeploymentEditor(Project project, AppEngineDeployable de
   @Override
   protected void resetEditorFrom(@NotNull AppEngineDeploymentConfiguration configuration) {
     commonConfig.resetEditorFrom(configuration);
+    commonConfig.getCompileEncodingTextField().setText(configuration.getCompileEncoding());
```

Comment:
if the fields live in the standard editor class like I mentioned in the other comment, then you wouldn't have to expose the UI fields with getters like this, they would just be part of this class